### PR TITLE
Added a user support to give po as a memberport for  access port in MLX

### PR DESCRIPTION
**Pull request by m-selvam**
UT results is uploaded in defect MLX-108

### CODE REVIEW

File: pyswitch/snmp/mlx/base/interface.py [hunk 1]
```diff
@@ -843,6 +843,10 @@ def acc_vlan(self, **kwargs):
         if not pyswitch.utilities.valid_vlan_id(vlan):
             raise InvalidVlanId("`name` must be between `1` and `4096`")
```
Comment (vijendarp) [inline pyswitch/snmp/mlx/base/interface.py]: Please change the valid vlan from 4096 to 4090

Comment (m-selvam) [reply pyswitch/snmp/mlx/base/interface.py]: valid_vlan_id is a common utility to check the vlan range from 1 to 4096, if user gives 4090,  cli-set will fail

Comment (vijendarp) [reply pyswitch/snmp/mlx/base/interface.py]: Previous comment was for the message you are logging as part of exception. For MLX we don't support above 4090

Comment (m-selvam) [reply pyswitch/snmp/mlx/base/interface.py]: Done

File: pyswitch/snmp/mlx/base/interface.py [hunk 2]
```diff
@@ -843,7 +843,11 @@ def acc_vlan(self, **kwargs):
 
         vlan = kwargs.pop('vlan')
         if not pyswitch.utilities.valid_vlan_id(vlan):
-            raise InvalidVlanId("`name` must be between `1` and `4096`")
+            raise InvalidVlanId("`name` must be between `1` and `4090`")
+
+        if int_type == 'port_channel':
+            name = self.get_lag_primary_port(name)
```
Comment (vijendarp) [inline pyswitch/snmp/mlx/base/interface.py]: What if there is no primary port configured for LAG? There can be a scenario where LAG is created but no member ports are added or member ports are added and no primary port is configured. You need to check if 'name' is not none.

Comment (m-selvam) [reply pyswitch/snmp/mlx/base/interface.py]: when there is no primary port, function get_lag_primary_port raises exception 
if primary_match is None or primary_match.group(1) is None:
            raise ValueError('primary port is not found for lag %s' % lag_id)

Comment (vijendarp) [reply pyswitch/snmp/mlx/base/interface.py]: The exception will not be raised as there is a bug in your get_lag_primary_port code. You are checking for 'None' whereas when there are no primary ports CLI returns 'none'.
=== LAG "t22" ID 1 (dynamic Not Deployed) ===
LAG Configuration:
   Ports:         
   Port Count:    0
   Primary Port:  none
   Trunk Type:    hash-based
   LACP Key:      100

Comment (vijendarp) [reply pyswitch/snmp/mlx/base/interface.py]: You will have to fix the API get_lag_primary_port () to raise the exception correctly when there is no primary port

**Pull request merged**